map.template.html

```diff
@@ -62,7 +62,7 @@ <h3>Edit</h3><br/>
       </div>
     </main>
 
-    <script type="text/javascript" src="svg-pan-zoom.js"></script>
-    <script type="text/javascript" src="../js/export.js"></script>
+    <script type="text/javascript" src="/topo/svg-pan-zoom.js"></script>
```

Comment:
Just out of curiosity, why 'topo'?

Reply:
topo as in topographic .. more specific to "maps"